debian/debathena-nsswitch-config.postinst

```diff
@@ -22,9 +22,16 @@ set -e
 
 case "$1" in
     configure)
-	if [ -x /var/lib/dpkg/info/libnss-mdns.postinst ]; then
-	    /var/lib/dpkg/info/libnss-mdns.postinst configure
-	fi
+	for postinst in \
+	    $(ldconfig -p \
+		  | sed -n 's/^\s*libnss_.*=> \(.*\.so\.2\)$/\1/p' \
+		  | xargs dpkg -S \
```

Comment:
Can library filenames have spaces (do we need -0)?

Reply:
In theory, they could have all kinds of horrible things, including `=>` and newlines. Since the input ultimately comes from an unparseable string puddle, trying to add null bytes and use `-0` for an intermediate stage doesn’t seem like it would really help. I suppose it’s easy enough to tighten the garbage regex and sprinkle in some `--`.